app/uk/gov/hmrc/apidocumentation/models/Documentation.scala

```diff
@@ -294,7 +279,7 @@ case class APIAvailability(endpointsEnabled: Boolean, access: APIAccess, loggedI
 
 case class VersionVisibility(privacy: APIAccessType.APIAccessType, loggedIn: Boolean, authorised: Boolean, isTrial: Option[Boolean] = None)
 
-case class Endpoint(
+case class ExtendedEndpoint(
```

Comment:
Were we going to move the func here into the library.  If not maybe we can put it somewhere else as a func taking an Endpoint and loose the ExtendedEndpoint